### DISCUSSION TITLE
Target a particular version of Insiders in my Github Action workflow.

### DISCUSSION THREAD

**Discussion by white-gorilla** [squidfunk/mkdocs-material#4882] [Q&A] [answered]
Hello,

I did a search on the forum, a [similar request was made](https://github.com/squidfunk/mkdocs-material/discussions/3374) but not for the Insiders version.
An [issue](https://github.com/squidfunk/mkdocs-material/issues/3363) has been issued and the [doc has been amended](https://github.com/squidfunk/mkdocs-material/commit/db5eee54aa7edb92140bfbdc276979db6263267f), but I don't see any answers.

If there is no question for the non-Insiders version:
```
pip install mkdocs-material==<version>
# or
docker pull squidfunk/mkdocs-material:<version>
```

I'm wondering how to do it for the Insiders version.
In theory, I should be able to do this, right?
```
run: pip install git+https://${GH_TOKEN}@github.com/squidfunk/mkdocs-material-insiders.git@<version>
```

**Comment by squidfunk** [comment, marked as answer]
Correct, releases are all tagged, so you could just use the syntax you mentioned. For the current latest version, this would be:

```
pip install git+https://${GH_TOKEN}@github.com/squidfunk/mkdocs-material-insiders.git@9.0.5-insiders-4.28.1
```
